server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/projection/cassandra/CassandraMailQueueModule.java

```diff
@@ -0,0 +1,65 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.queue.rabbitmq.projection.cassandra;
+
+import static com.datastax.driver.core.DataType.bigint;
+import static com.datastax.driver.core.DataType.blob;
+import static com.datastax.driver.core.DataType.list;
+import static com.datastax.driver.core.DataType.map;
+import static com.datastax.driver.core.DataType.smallint;
+import static com.datastax.driver.core.DataType.text;
+import static com.datastax.driver.core.DataType.timestamp;
+import static com.datastax.driver.core.schemabuilder.SchemaBuilder.frozen;
+
+import org.apache.james.backends.cassandra.components.CassandraModule;
+
+public interface CassandraMailQueueModule {
+
+    CassandraModule MODULE = CassandraModule
+        .table(EnqueuedMailsTable.TABLE_NAME)
+        .comment("store enqueued mail")
+        .options(options -> options)
+        .statement(statement -> statement
+            .addPartitionKey(EnqueuedMailsTable.QUEUE_NAME, text())
+            .addPartitionKey(EnqueuedMailsTable.TIME_RANGE_START, timestamp())
+            .addPartitionKey(EnqueuedMailsTable.BUCKET_ID, smallint())
+            .addPartitionKey(EnqueuedMailsTable.MAIL_KEY, text())
+            .addColumn(EnqueuedMailsTable.MESSAGE_SIZE, bigint())
```

Comment:
From my experience `MimeMessage::getSize` is sometime buggy (thanks java-mail) and leads to storage errors.

We don't want our queue to "swallow" such messages, and this information is not needed as size is computed using the blob by MailImpl.

I would suggest to remove that field...